### PR TITLE
Update readme with user profile

### PR DESCRIPTION
**Pull request by Zzmobie**

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -96,14 +96,14 @@ This command takes no arguments.
 ==== Adding a person: `add`
 
 Adds a person to the address book. +
-Format: `add n/NAME [p/PHONE_NUMBER]... [e/EMAIL]... [a/ADDRESS]... [fb/FACEBOOK] [tw/TWITTER] [ig/INSTAGRAM] [t/TAG]...` +
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [fb/FACEBOOK] [tw/TWITTER] [ig/INSTAGRAM] [t/TAG]...` +
 Alias: `a`
 
 [TIP]
-A person can have any number of tags, phone numbers, emails or addresses (including 0) (Coming v2.0)
+A person can have any number of tags, phone numbers, emails or addresses (including 0) (Not implemented yet. Coming v2.0)
```
Comment (donjar) [inline docs/UserGuide.adoc]: Let's change this to `Coming in v2.0` to make it consistent with the below as well

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -585,6 +596,7 @@ e.g.`remark 4 r/new remark`
 e.g. `socialmedia 1 facebook`
 * *Birthday* : `birthday`
 * *Font size* : `size AMOUNT` +
+* *Access display* : `accessdisplay TYPE` +
```
Comment (donjar) [inline docs/UserGuide.adoc]: `Toggle access display`?

**Review by donjar — APPROVED** [review]
LGTM but looks like you have trailing whitespace:

ERROR:README.adoc:19: trailing whitespace.

File: docs/UserGuide.adoc (hunk 1 above)
Comment (Zzmobie) [reply docs/UserGuide.adoc]: I changed it to make it more explicit what features haven't been included yet. I think I changed it all throughout? Unless I missed something.

**Pull request merged**